packages/did/README.md

```diff
@@ -96,15 +104,16 @@ const lightDID = new LightDidDetails({
     publicKey: encryptionKeyPublicDetails.publicKey,
     type: DemoKeystore.getKeypairTypeForAlg(encryptionKeyPublicDetails.alg),
   },
+  serviceEndpoints,
 })
 
-// Will print `did:kilt:light:014sxSYXakw1ZXBymzT9t3Yw91mUaqKST5bFUEjGEpvkTuckar:oWFlomlwdWJsaWNLZXlYILu7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7ZHR5cGVmeDI1NTE5`.
+// Will print `did:kilt:light:014sxSYXakw1ZXBymzT9t3Yw91mUaqKST5bFUEjGEpvkTuckar:omFlomlwdWJsaWNLZXlYILu7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7u7ZHR5cGVmeDI1NTE5YXOBo2JpZHNteS1zZXJ2aWNlLWVuZHBvaW50ZXR5cGVzgnZDb2xsYXRvckNyZWRlbnRpYWxUeXBlbVNvY2lhbEtZQ1R5cGVkdXJsc4J1aHR0cHM6Ly9teV9kb21haW4ub3JnbXJhbmRvbV9kb21haW4`.
 console.log(lightDID.did)
 ```
 
 ## Full DIDs
 
-As mentioned above, the creation of a full DID requires interaction with the KILT blockchain. Therefore, it is necessary for the DID creation operation to be submitted by a KILT address with enough funds to pay the transaction fees and the required deposit (0,1 KILT).
+As mentioned above, the creation of a full DID requires interaction with the KILT blockchain. Therefore, it is necessary for the DID creation operation to be submitted by a KILT address with enough funds to pay the transaction fees and the required deposit (2 KILTs).
```

Comment:
Q: The kilt amounts might be subject to change, so I think about removing them here to not have to keep track of it here?

Reply:
https://github.com/KILTprotocol/sdk-js/pull/428/commits/3d156cfa8cb5e2b14b593e11b224c6a0315b3688

Reply:
There is a second place which talks about a specific token amount: https://github.com/KILTprotocol/sdk-js/pull/428/files/bfc0773f65e9742d7ca2ce1333800857ffe8d273#diff-aefa9d86a34d118f963142ae95b7e6f36a01857bc2e97ad5905599d909fdf7c8R365 let's remove it too

Reply:
https://github.com/KILTprotocol/sdk-js/pull/428/commits/fb6d8dbda40755622136ebdbb6f9753d4fa54a95